### PR TITLE
Added ssmis.yaml with QC

### PR DESCRIPTION
**Pull request by azadeh-gh**
Added two ssmis yaml files in config and testing. 
The yaml file has obs filters to assign obs errors. 
SSMIS Obs Validation checked: 
 - HofX computation using GSI produced GeoVaLs
 - QC count checks (and YAML changes) using GSI produced GeoVaLs
 - Obs errors using GSI produced GeoVaLs
 - HofX computation in FV3-JEDI

### CODE REVIEW

File: parm/atm/obs/config/ssmis_f17.yaml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    engine:
+      type: H5File
+      obsfile: $(OBS_DIR)/$(OBS_PREFIX)ssmis_f17.$(OBS_DATE).nc4
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: $(DIAG_DIR)/diag_ssmis_f17_$(OBS_DATE).nc4
+  simulated variables: [brightness_temperature]
+  channels: 1-24
+geovals:
```
Comment (CoryMartin-NOAA) [inline parm/atm/obs/config/ssmis_f17.yaml]: no geovals for the `config` yamls

File: parm/atm/obs/config/ssmis_f17.yaml [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  test variables:
+  - name: ChannelUseflagCheckRad@ObsFunction
+    channels: 1-24
+    options:
+      channels: 1-24
+      use_flag: [ 1, -1, -1, -1, 1, 1, 1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 1]
+  minvalue: 1.0e-12
+  action:
+    name: reject
+passedBenchmark: 440   #(m_unittest)
```
Comment (CoryMartin-NOAA) [inline parm/atm/obs/config/ssmis_f17.yaml]: don't need this end line since it is just for 'tests'

**Review by CoryMartin-NOAA — COMMENTED** [review]
Couple of small things

**Review by nicholasesposito — DISMISSED** [review]
Looks good to me

**Comment by CoryMartin-NOAA** [comment]
Is the second SSMIS going to be added in this PR or as part of a separate one?

**Comment by azadeh-gh** [comment]
the second SSMIS going to be added in this PR

**Review by emilyhcliu — APPROVED** [review]
Having all obs errors assigned in one filter with a vector is good.  
The changes look good.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**